application/common/mail/password-changed.text.php

```diff
@@ -0,0 +1,84 @@
+ * @var string $locked
+ * @var string $lastChangedUtc
+ * @var string $lastSyncedUtc
+ * @var string $lastLoginUtc
+ * @var string $passwordExpiresUtc
+ * @var string $emailSignature
+ * @var string $helpCenterUrl
+ * @var string $idpDisplayName
+ * @var string $passwordForgotUrl
+ * @var string $passwordProfileUrl
+ * @var string $supportEmail
+ * @var string $supportName
+ * @var bool   $isMfaEnabled
+ */
+?>
+Dear <?=yHtml::encode($displayName)?>,
+
+The password for your <?=yHtml::encode($idpDisplayName)?> account has been changed. If you did not make this change
+please contact <?=yHtml::encode($supportName)?> at <?=yHtml::encode($supportEmail)?> immediately to let us know.
+
+Please remember that this account will be your primary means for logging into many corporate
+applications. It is also important to note that this account (username and password) is unique and will not be kept
+in sync with any other accounts you have.
+
+Password changed on: <?=yHtml::encode(MySqlDateTime::formatDateForHumans($lastChangedUtc))?>
+Password expires on: <?=yHtml::encode(MySqlDateTime::formatDateForHumans($passwordExpiresUtc))?>
+
+If you have not already done so, it is highly recommended that you configure recovery methods for
+the potential event that you forget your password. You can reset your password using your email address,
+<?=yHtml::encode($email)?>, but you can also add other addresses and even phone numbers for SMS verification.
+
+Instructions to add recovery methods:
+-------------------------------------
+1. Go to <?=yHtml::encode($passwordProfileUrl)?>.
+2. Click the "Add" button next to Recovery Methods.
+3. Select the option for either an Email or Phone recovery method.
+4. Enter the email address or phone number you wish to use and click "Send Code"
+5. If you entered an email address, check the inbox for that email address for a new email and retrieve the code
+   from that email.
+6. If you entered a phone number, watch for a text message or phone call and take note of the code you receive.
+7. Enter the verification code into the form on your screen and click "Verify".
+
+<?php
+if ( ! $isMfaEnabled) {
+    ?>
+    
+Enable 2-Step Verification for enhanced security
+================================================
+2-Step Verification can help keep bad guys out, even if they have your password. With 2-Step Verification, you'll
+protect your account with something you know (your password) and something you have (your phone or Security Key).
+Setup is easy and with the option to remember your computer for 30 days at a time, you’ll only need to use the second
+step every month or so, but anyone trying to hack into your account would need both factors.
+    
+Instructions to set up 2-Step Verification:
+-------------------------------------------
+1. Go to <?=yHtml::encode($passwordProfileUrl)?>
+2. Under 2-Step Verification, set up the options that suite you best (USB Security Key, Smartphone App, and/or
+   Printable Backup Codes)
```

Comment:
As above, should this be `Printable Codes`?

Reply:
fixed